pkg/controller/pxc/controller.go

```diff
@@ -254,14 +261,36 @@ func (r *ReconcilePerconaXtraDBCluster) Reconcile(request reconcile.Request) (re
 	return rr, nil
 }
 
+func (r *ReconcilePerconaXtraDBCluster) operatorPod(cr *api.PerconaXtraDBCluster) (corev1.Pod, error) {
+	operatorPod := corev1.Pod{}
+	err := r.client.Get(context.TODO(),
+		types.NamespacedName{
+			Namespace: cr.Namespace,
+			Name:      os.Getenv("HOSTNAME"),
+		},
+		&operatorPod,
+	)
+	if err != nil {
+		return operatorPod, err
+	}
+
+	return operatorPod, nil
+}
+
 func (r *ReconcilePerconaXtraDBCluster) deploy(cr *api.PerconaXtraDBCluster) error {
 	stsApp := statefulset.NewNode(cr)
 	err := r.reconcileConfigMap(cr)
 	if err != nil {
 		return err
 	}
 
-	nodeSet, err := pxc.StatefulSet(stsApp, cr.Spec.PXC, cr)
+	operatorPod, err := r.operatorPod(cr)
+	if err != nil {
+		return errors.Wrap(err, "get operator deployment")
+	}
+
+	epInit := statefulset.EntrypointInitContainer(operatorPod.Spec.Containers[0].Image)
```

Comment:
epInit should be added only for 1.5.0

Reply:
https://github.com/percona/percona-xtradb-cluster-operator/pull/363/commits/2d0ba5f05d6c11f56579d494f1437ff396c95c45